pom.xml

```diff
@@ -54,6 +61,9 @@
         <dep.oracle.version>19.3.0.0</dep.oracle.version>
         <dep.drift.version>1.14</dep.drift.version>
         <dep.tempto.version>180</dep.tempto.version>
+        <!-- coral version for reading hive views -->
+        <dep.coral.version>1.0.5</dep.coral.version>
+        <dep.calcite.version>1.21.1.1</dep.calcite.version>
```

Comment:
No need for a property here since it's only used for one dependency. We only add properties for versions that need to be the same across multiple dependencies.

Reply:
Removed.